dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -1,62 +1,362 @@
+from __future__ import annotations
+
+import itertools
+import logging
 import re
+from abc import ABC, abstractmethod
+from datetime import datetime
+from functools import lru_cache
 from itertools import chain
-from typing import Iterator
+from pathlib import Path
+from typing import Any, Iterator
 
+from dissect.target import Target
 from dissect.target.exceptions import UnsupportedPluginError
-from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.helpers.fsutil import open_decompress
+from dissect.target.helpers.record import DynamicDescriptor, TargetRecordDescriptor
 from dissect.target.helpers.utils import year_rollover_helper
-from dissect.target.plugin import Plugin, export
-
-AuthLogRecord = TargetRecordDescriptor(
-    "linux/log/auth",
-    [
-        ("datetime", "ts"),
-        ("string", "message"),
-        ("path", "source"),
-    ],
+from dissect.target.plugin import Plugin, alias, export
+
+log = logging.getLogger(__name__)
+
+RE_TS = re.compile(r"^[A-Za-z]{3}\s*\d{1,2}\s\d{1,2}:\d{2}:\d{2}")
+RE_TS_ISO = re.compile(r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}\+\d{2}:\d{2}")
+RE_LINE = re.compile(
+    r"""\d{2}:\d{2}\s                           # First match on the similar ending of the different timestamps
+        (?P<hostname>\S+)\s                     # The hostname
+        (?P<service>\S+?)(\[(?P<pid>\d+)\])?:   # The service with optionally the PID between brackets
+        \s*(?P<message>.+?)\s*$                 # The log message stripped from spaces left and right""",
+    re.X,
```

Comment:
```suggestion
    r"""
    \d{2}:\d{2}\s                           # First match on the similar ending of the different timestamps
    (?P<hostname>\S+)\s                     # The hostname
    (?P<service>\S+?)(\[(?P<pid>\d+)\])?:   # The service with optionally the PID between brackets
    \s*(?P<message>.+?)\s*$                 # The log message stripped from spaces left and right
    """,
    re.VERBOSE,
```
Can you format them all to have the `r"""` and closing `"""` on separate lines? Also use `re.VERBOSE` everywhere instead of `re.X`.